cbmackintosh/fast-command
# Feature/redux login integration

**Clearly and concisely describe the feature.**
- Reworks the login system on the frontend to incorporate Redux Toolkit and global state management

**Is this a feature or refactor?**
- Refactor

**Problems Encountered & Resulting Solutions**
- Getting the signup to automatically redirect to dashboard upon successful submission was tricky. Ended up reworking the pervious solution, which involves passing in props to the Signup component and pushing "/" to the history attribute.

**Where should the reviewer start?**
- The entire component architecture was completely reworked. Also see changes to the global store and the newly created api-calls file. BrowserRouter was moved from App component to index.js

**How should this be tested?**
- Clone down this repo and try logging in or signing up for a new account. Application should function the same as before. Check Redux dev tools to verify that global state updates as expected.

**Steps To Take**
- Next I'll probably create a way to add and search contacts in the user's personal database.
